api/flask_app.py

```diff
@@ -0,0 +1,50 @@
+
+import face_recognition
+import cv2
+import datetime
+
+import generator
+
+from flask import Flask, render_template, make_response, url_for, Response
+from flask_restful import Resource, Api
+
+app = Flask(__name__)
+api = Api(app)
+
+
+date = datetime.date.today()
+
+''' For now we are using dummy data but then we will get it from the video-chat or database. '''
+interview_data = [
+    {
+        'interviewer': 'Epam Mahager',
+        'interviewee': 'Firstname Lastname',
+        'agenda': 'Data Analyst position',
+        'date_time': '{:%B %d, %Y}\n'.format(date),
+        'image_source': '/static/default.jpg',
+        'video_source': '/static/IMG_0909.MOV'
```

Comment:
> To be able to use this API for processing video streams I suppose your code should support behavior:
> 
> * API should accept frames instead video, because when video stream/call is in progress it's just a steam of frames, not a video file encoded via some codec.
> * For demo better to prepare some stand alone script which will open video file, send it's content frame by frame to the API and show processed frames on the screen
> 
> Does it sounds reasonable for you?

> Ok, I got you. Yes it sounds good, I'll do that.

Reply:
> We may merge this PR without doing changes described above and do then in scope of other issue. Please create appropriate issue in project `Issues` if you prefer this way. And we may discuss how to make several running options: API that support images only, UI with video upload, UI with web camera capture.

This sounds good as well, I'll think it over and we'll discuss it during our meeting and make a decision afterwards.